app/controllers/customers.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  actions: {
+    deleteCustomer() {
+      this.get('model')
+        .destroyRecord();
```

Comment:
any reason this isn't on one line like https://github.com/artisan-tattoo/assistant-frontend/pull/30/files#diff-003fa161573a6c036fc4cf470f9b4261R6?